lte/cloud/go/services/lte/servicers/builder_servicer_test.go

```diff
@@ -831,6 +836,134 @@ func TestBuilder_BuildUnmanagedEnbConfig(t *testing.T) {
 	assert.Equal(t, expected, actual)
 }
 
+func TestBuilder_Build_MMEPool(t *testing.T) {
+	assert.NoError(t, plugin.RegisterPluginForTests(t, &pluginimpl.BaseOrchestratorPlugin{}))
+	assert.NoError(t, plugin.RegisterPluginForTests(t, &lte_plugin.LteOrchestratorPlugin{}))
+	lte_test_init.StartTestService(t)
+
+	nw := configurator.Network{
+		ID: "n1",
+		Configs: map[string]interface{}{
+			lte.CellularNetworkConfigType: lte_models.NewDefaultTDDNetworkConfig(),
+			orc8r.DnsdNetworkType: &models.NetworkDNSConfig{
+				EnableCaching: swag.Bool(true),
+			},
+		},
+	}
+	gw := configurator.NetworkEntity{
+		Type: orc8r.MagmadGatewayType, Key: "gw1",
+		Associations: []storage.TypeAndKey{
+			{Type: lte.CellularGatewayEntityType, Key: "gw1"},
+		},
+	}
+	lteGatewayPool := configurator.NetworkEntity{
+		Type: lte.CellularGatewayPoolEntityType, Key: "pool1",
+		Config: &lte_models.CellularGatewayPoolConfigs{
+			MmeGroupID: 2,
+		},
+	}
+	lteGatewayConfigs := newDefaultGatewayConfig()
+	lteGatewayConfigs.Pooling = lte_models.CellularGatewayPoolRecords{
+		{
+			GatewayPoolID: "pool1",
+			MmeCode: 1,
```

Comment:
Can you make this MmeCode value != 1, just to have one test case where the value isn't =1